gobblin-core-base/src/main/java/org/apache/gobblin/source/extractor/extract/FlushingExtractor.java

```diff
@@ -298,6 +298,14 @@ protected void publishTaskOutput() throws IOException {
     this.flushPublisher.get().publish(Collections.singletonList(workUnitState));
   }
 
+  @Override
+  public void shutdown() {
```

Comment:
What's the reason super.shutdown is not called, is this intentional ?

Reply:
Yeah, it's a newly added method which will be called in kafkaStreamingExtractor.shutdown(), and the kafkaStreamingExtractor.shutdown() method did not call super.shutdown(), so I did not add it here as well. Also super.shutdown is meaning to check it's a decorator, which is not applied to this case as well.